README.md

```diff
@@ -289,6 +293,7 @@ Available types:
 - `isVoid` <- whether type is Void (`Void` or `()`)
 - `isTuple` <- whether given type is a tuple
 - `tuple.elements` <- if given type is a tuple returns its elements information (*TupleType.Element*)
+- `attributes` <- type attributes, i.e. `typeName.attributes.escaping`, or `typeName.escaping`
```

Comment:
I think we are better off wrapping it in `attributes` rather than exposing it via shorthand. I'd rather avoid the possibility of nameclash